libs/libglog/Makefile

```diff
@@ -0,0 +1,54 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=glog
+PKG_VERSION:=0.3.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/google/glog/archive
+PKG_MD5SUM:=df92e05c9d02504fb96674bc776a41cb
+
+PKG_FIXUP:=autoreconf
+PKG_USE_MIPS16:=0
```

Comment:
What error message do you get with mips16? This is just a question, if you get one this is ok.